[2024-12-06T21:24:26.979+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-12-06T21:24:26.994+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: homework_zone_dag.delete_dataset manual__2024-12-06T21:24:01.598190+00:00 [queued]>
[2024-12-06T21:24:27.001+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: homework_zone_dag.delete_dataset manual__2024-12-06T21:24:01.598190+00:00 [queued]>
[2024-12-06T21:24:27.001+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2024-12-06T21:24:27.011+0000] {taskinstance.py:2889} INFO - Executing <Task(BashOperator): delete_dataset> on 2024-12-06 21:24:01.598190+00:00
[2024-12-06T21:24:27.031+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=5182) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-12-06T21:24:27.033+0000] {standard_task_runner.py:72} INFO - Started process 5183 to run task
[2024-12-06T21:24:27.032+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'homework_zone_dag', 'delete_dataset', 'manual__2024-12-06T21:24:01.598190+00:00', '--job-id', '143', '--raw', '--subdir', 'DAGS_FOLDER/homework_zone_dag.py', '--cfg-path', '/tmp/tmp70hbymw1']
[2024-12-06T21:24:27.036+0000] {standard_task_runner.py:105} INFO - Job 143: Subtask delete_dataset
[2024-12-06T21:24:27.072+0000] {task_command.py:467} INFO - Running <TaskInstance: homework_zone_dag.delete_dataset manual__2024-12-06T21:24:01.598190+00:00 [running]> on host 39ecaeffba2c
[2024-12-06T21:24:27.115+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='homework_zone_dag' AIRFLOW_CTX_TASK_ID='delete_dataset' AIRFLOW_CTX_EXECUTION_DATE='2024-12-06T21:24:01.598190+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-12-06T21:24:01.598190+00:00'
[2024-12-06T21:24:27.116+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-12-06T21:24:27.129+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2024-12-06T21:24:27.130+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'rm /opt/***/zones.csv']
[2024-12-06T21:24:27.140+0000] {subprocess.py:99} INFO - Output:
[2024-12-06T21:24:27.152+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2024-12-06T21:24:27.170+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-12-06T21:24:27.171+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=homework_zone_dag, task_id=delete_dataset, run_id=manual__2024-12-06T21:24:01.598190+00:00, execution_date=20241206T212401, start_date=20241206T212426, end_date=20241206T212427
[2024-12-06T21:24:27.220+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-12-06T21:24:27.231+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-12-06T21:24:27.232+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
